### PR TITLE
Refactor permissions switch fallthrough

### PR DESCRIPTION
**Pull request by jweak**

### CODE REVIEW

File: src/Permissions.ts [hunk 1]
```diff
@@ -56,56 +56,73 @@ export class Permissions {
...
       case Actions.GetBillingPageToken:
+        return isAdminOrOwner;
+
       case Actions.DeleteInvitationDomain:
+        return isAdminOrOwner;
+
       default:
-        canI = [Roles.Owner, Roles.Admin].includes(this.getRole(forSpace, forUserId));
-        break;
+        return isAdminOrOwner;
```
Comment (panuhorsmalahti) [inline src/Permissions.ts:124]: Maybe default to false, which would expose bugs more easy. (e.g. missing action)

File: src/Permissions.ts [hunk 2]
```diff
@@ -56,56 +56,73 @@ export class Permissions {
       invitationIdsCreatedByUserId: string[];
     }
   ) {
-    let canI = false;
 
+    const role = this.getRole(forSpace, forUserId);
+    const isAdminOrOwner = [Roles.Owner, Roles.Admin].includes(role);
+
+    const canPatchOrRevokeInvitation = () => {
```
Comment (panuhorsmalahti) [inline src/Permissions.ts:63]: Move this to a class method

File: src/Permissions.ts [hunk 3]
```diff
@@ -56,56 +56,73 @@ export class Permissions {
...
       case Actions.ChangeSpacePlan:
-      case Actions.DeleteSpace:
-        canI = forSpace.kind !== "Personal" && this.getRole(forSpace, forUserId) === Roles.Owner;
-        break;
-      case Actions.PatchInvitation:
-      case Actions.RevokeInvitation: {
-        if ([Roles.Owner, Roles.Admin].includes(this.getRole(forSpace, forUserId))) {
-          canI = true;
-        }
+        return role === Roles.Owner
```
Comment (panuhorsmalahti) [inline src/Permissions.ts:85]: `;` missing

**Review by panuhorsmalahti — CHANGES_REQUESTED** [review]

File: src/Permissions.ts (hunk 1 above)
Comment (jweak) [reply src/Permissions.ts]: good point, done

File: src/Permissions.ts (hunk 3 above)
Comment (jweak) [reply src/Permissions.ts]: yeah, vs code doesn't fix lint on this repo...

File: src/Permissions.ts (hunk 2 above)
Comment (jweak) [reply src/Permissions.ts]: good point. fixt

**Review by panuhorsmalahti — APPROVED** [review]

**Pull request merged**